src/Zonotope.jl

```diff
@@ -684,3 +685,36 @@ function translate(Z::Zonotope{N}, v::AbstractVector{N}; share::Bool=false
     generators = share ? Z.generators : copy(Z.generators)
     return Zonotope(c, generators)
 end
+
+"""
+    overapproximate(Z::Zonotope)
+
+Reduce the order of a given Zonotope to 1
```

Comment:
```suggestion
Reduce the order of a given Zonotope to 1.
```
Given that description, maybe rename to `reduce_order`? We already have such a function, so maybe integrate there with the argument `order == 1`?

Reply:
This new function converts the set representation to a parallelotope. It was marked as WIP, becaue the parallelotope set type has not been added yet.

Reply:
The existing function to reduce the order of the zonotope is meant to return another zonotope.

Reply:
> This new function converts the set representation to a parallelotope.

Then this comment should be changed.

Reply:
> This new function converts the set representation to a parallelotope.

Then this PR does not close #1328.

Reply:
Yes. We are haven't yet defined the path to implementing the methods in [1], but one way to proceed is to make this PR affect zonotope -> paralellotope only, leaving the conversion for order greater than 1 (polytope) for another PR.